### DISCUSSION TITLE
OAuth Security Application GitHub denied by BSIT for Snowflake -sys 3rd party deleted

### DISCUSSION THREAD

**Discussion by Snowflake-sys** [community/events#66] [Cloud]
After that we will SharePoint and do an intake of the application and process full public disclosures of social media including, PP in bridge to SD. 3rd party public information will be collected on each email and post by appointment application not on-line.

**Comment by jricciardi** [comment]
Hi there - we want to keep the conversation on topic around GitHub Universe. If you'd like to ask a question, consider checking out the wider [GitHub Community Discussions](https://github.com/orgs/community/discussions). If you'd like to reach out to GitHub Support, you can [find more information here.](https://support.github.com/)

Happy Universe!
